cfgov/v1/atomic_elements/organisms.py

```diff
@@ -397,6 +416,13 @@ def make_created_value(self, instance, value):
         help_text='Stack the table columns on mobile.'
     )
 
+    empty_table_msg = blocks.CharBlock(
+        label='No Jobs Message',
+        required=False,
+        default='There are no current openings at this time.',
+        help_text='Message to display if there are no jobs.'
```

Comment:
Because this is a generic `ModelTable`, can we make these defaults/labels not jobs-specific?